subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationSpec.groovy

```diff
@@ -765,6 +764,35 @@ class DefaultConfigurationSpec extends Specification {
         assert copiedConfiguration.extendsFrom.empty
     }
 
+    def "Calls resolve actions and closures from original configuration when copied configuration is resolved"() {
```

Comment:
Can you add an `@Issue` annotation to indicate the relevant issue? I would like like this: `@Issue("gradle/gradle#1567")`.